lib/internal/crypto/webidl.js

```diff
@@ -194,7 +195,7 @@ converters.object = (V, opts) => {
 };
 
 function isNonSharedArrayBuffer(V) {
-  return ObjectPrototypeIsPrototypeOf(ArrayBufferPrototype, V);
+  return isArrayBuffer(V);
```

Comment:
Alternatively this could be:

```js
const isNonSharedArrayBuffer = isArrayBuffer;
```

Also, should `isSharedArrayBuffer(...)` also be similarly updated here?

Reply:
I would actually just remove the method and use `isArrayBuffer()` directly instead. It should be clear that it's not shared.

Reply:
I'm confused - a SharedArrayBuffer is also an ArrayBuffer, in terms of internal slots - what actual check does `isArrayBuffer` perform?

Reply:
[`util.types.isArrayBuffer()`](https://github.com/nodejs/node/blob/91d8a524ada001103a2d1c6825ca17b8393c183f/src/node_types.cc#L44-L50) invokes `v8::Value::isArrayBuffer`, which returns false if it is a SharedArrayBuffer: https://source.chromium.org/chromium/chromium/src/+/main:v8/src/api/api.cc;l=3534-3538?q=Value::IsArrayBuffer.